src/useQuery.ts

```diff
@@ -145,6 +151,16 @@ export function useQuery<TData = any, TVariables = OperationVariables>(
     updateQuery: observableQuery.updateQuery.bind(observableQuery),
   };
 
+  if (skip) {
+    // Taken from https://github.com/apollographql/react-apollo/blob/5cb63b3625ce5e4a3d3e4ba132eaec2a38ef5d90/src/Query.tsx#L376-L381
+    return {
+      ...helpers,
```

Comment:
I don't think generic helpers should be returned here (or even created before) - they could be no-functional. I'd create a separate version of helpers which always throw. Please look at https://gist.github.com/trojanowski/f7e4e75659d2de1166274ad8f0d06bb4

Reply:
How does that work in Apollo? Are they stubbing those helpers too? Also let's please use DEV-only invariant, throwing an error like that can get ugly.

Reply:
Done, I add test cases.

Reply:
Oops, it's already merged 😅

Reply:
> Done, I add test cases.

@umidbekkarimov Could you please create a PR with those changes? Maybe it's worth a further discussion. What's your opinion about it?

Reply:
@trojanowski I think it's better to keep things like in react-apollo. We just will expose api of observable query, but developers are responsible for their usage. E.g (https://github.com/trojanowski/react-apollo-hooks/pull/38#issuecomment-449369084):

```js
const { accessToken } = useAuthToken();
const { data, errors, loading, refetch } = useQuery(MeQuery, {
  skip: !accessToken
});

useEffect(
  () => {
    if (accessToken) { // Otherwise it will produce http error.
      refetch(); 
    }
  },
  [accessToken]
);
```